services/dataset/index.ts

```diff
@@ -0,0 +1,68 @@
+import WRISerializer from 'wri-json-api-serializer';
+
+import DATASETS from 'data/datasets.json';
+// import { WRIAPI } from 'lib/axios';
+import { Dataset } from 'types';
+
+// utils
+
+// API docs: https://resource-watch.github.io/doc-api/index-rw.html#dataset
+
+/**
+ * Fetches datasets according to params.
+ * Check out the API docs for this endpoint {@link https://resource-watch.github.io/doc-api/index-rw.html#getting-all-datasets|here}
+ * @param {Object} params Request parameters.
+ * @param {Object} headers Request headers.
+ * @param {boolean} _meta Boolean flag indicating whether the meta object should
+ * @returns {Array} Array of serialized datasets.
+ * be included in the response or not.
+ */
+export function fetchDatasets(): Promise<Dataset[]> {
+  // params = {},
+  // headers = {},
+  // _meta = false
+  // todo: remove this line when datasets are in the API.
+  return Promise.resolve([WRISerializer(DATASETS)]);
+}
+// const newParams = {
```

Comment:
I know why you have this commented but I prefer to create another file or function